docs/reference/db/configuration.md

```diff
@@ -1,577 +0,0 @@
-# Configuration
```

Comment:
Despite the look, this file was not duplicated, only some part of it. This should contain the section that are not duplicated.